.devcontainer/setup_container.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Link root to new batman user
+rm /home/batman/.bashrc
+ln -s /root/.pyenv      /home/batman/.pyenv
```

Comment:
with this batman doesn't have permissions to install new packages. needs to be chmoded/chowned